plugins/inputs/vault/README.md

```diff
@@ -0,0 +1,35 @@
+# Hashicorp Vault Input Plugin
+
+The Vault plugin could grab metrics from every Vault agent of the cluster. Telegraf may be present in every node and connect to the agent locally. In this case should be something like `http://127.0.0.1:8200`.
+
+> Tested on vault 1.8.5
+
+## Configuration
+
+```toml
+[[inputs.vault]]
+  ## URL for the vault agent
+  # url = "http://127.0.0.1:8200"
+
+  ## Use auth token for authorization.
+  ## Vault token configuration is mandatory.
+  ## If both are empty or both are set, an error is thrown.
+  # vault_token = "/path/to/auth/token"
+  ## OR
+  vault_token_string = "s.CDDrgg5zPv5ssI0Z2P4qxJj2"
```

Comment:
Maybe name those `token` and `token_file` to be more clear. No need to prepend `vault_` as it is under that section anyway.